tests/fulldecay/test_fulldecay.py

```diff
@@ -0,0 +1,89 @@
+from example_decay_chains import *  # TODO remove * since it is bad practice?
+from numpy.testing import assert_almost_equal
+
+from phasespace.fulldecay import FullDecay
+
+
+def check_norm(full_decay: FullDecay, **kwargs) -> list[tuple]:
+    """Checks whether the normalize_weights argument works for FullDecay.generate.

```

Comment:
This works, but if it fails, the assert won't print an informative message (since it is treated as a normal python assert). You can register an assert rewrite, as described here: https://stackoverflow.com/questions/41522767/pytest-assert-introspection-in-helper-function

Reply:
Fixed in 968cf0b7a61259e1042ba6886c8941403dbf7e88